### ISSUE TITLE
API for contacts does not support V2

### ISSUE THREAD

**Issue by CristianCardosoA** [messagebird/messagebird-nodejs#63]
At the moment that I send a message via template what's app, the returning contact id is available only on https://contacts.messagebird.com/v2/contacts

If I want to read a contact using this SDK, the contact id does not exist because the contact only exists on the V2

**Comment by mercteil** [comment]
Yes, same issue. And it seems your comment is now 2 years old... hmmm
